config/flags_internal.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package config
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+const (
+	// domains flag
+	domainsFlagID = "domains"
+)
+
+// SetupAllUpdateManagerFlags adds all flags for the configuration of the update manager
+func SetupAllUpdateManagerFlags(flagSet *flag.FlagSet, cfg *Config) {
+	SetupFlags(flagSet, cfg.BaseConfig)
+
+	flagSet.String(domainsFlagID, "", "Specify a comma-separated list of domains handled by the update manager")
```

Comment:
Can you use custom flags for this one?

Reply:
Maybe we can modify flag management later, but for the moment this works fine.